scAR/main/_activation_functions.py

```diff
@@ -1,20 +1,22 @@
+"""Activation functions"""
 # -*- coding: utf-8 -*-
 
-import torch
```

Comment:
pylint: unused import
@CaibinSh Can you please make sure that this does not raise any issue?